docs/index.rst

```diff
@@ -6,7 +6,12 @@
    contain the root `toctree` directive.
 
 Welcome to disnake
-===========================
+==================
+
+.. toctree::
+    :hidden:
+
+    api
```

Comment:
what is this for?

if this is to silence a warning, don't do this. Instead, put `:orphan:` on that page, and sphinx won't complain about not finding it anywhere.

Reply:
Thanks! Done in [`d15ddd7`](https://github.com/DisnakeDev/disnake/pull/722/commits/d15ddd70e20c611070a3445379763a80b07a8c13)